server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/InternalAutoDateHistogram.java.orig

```diff
@@ -0,0 +1,645 @@
+/*
```

Comment:
this file is probably leftover, no?

Reply:
Oooof. I don't know how this got added/committed.  My git-fu is weak today apparently